### PR TITLE
docs: translation for input in react dom

### LINKED ISSUE #423: `<input>`
Page: https://id.react.dev/reference/react-dom/components/input

Assignee: @abidra

### PR DESCRIPTION
**Pull request by abidra**
Closes #423

Translation to Bahasa for Input in React Dom.

Reference: #330

### CODE REVIEW

File: src/content/reference/react-dom/components/input.md
Comment (eriansha) [inline src/content/reference/react-dom/components/input.md:589]: i think "it" in this context is referring to the `value`, not the component. In this case, "komponen" should be replaced with "nilai"

Comment (abidra) [reply src/content/reference/react-dom/components/input.md:589]: yes correct, thanks for the feedback @eriansha!

File: src/content/reference/react-dom/components/input.md
Comment (FelixFern) [inline src/content/reference/react-dom/components/input.md:7]: "browser" should be written as "peramban" according to the https://github.com/reactjs/id.react.dev/wiki/Glosarium

File: src/content/reference/react-dom/components/input.md
Comment (FelixFern) [inline src/content/reference/react-dom/components/input.md:23]: "browser" should be written as "peramban" according to the https://github.com/reactjs/id.react.dev/wiki/Glosarium

File: src/content/reference/react-dom/components/input.md
Comment (FelixFern) [inline src/content/reference/react-dom/components/input.md:151]: "browser" should be written as "peramban" according to the https://github.com/reactjs/id.react.dev/wiki/Glosarium

File: src/content/reference/react-dom/components/input.md
Comment (r17x) [inline src/content/reference/react-dom/components/input.md:23]:
```suggestion
Untuk menampilkan sebuah masukan, *render* komponen [bawaan peramban `<input>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input).
```

**Review by r17x — DISMISSED** [review]
All words like "render" need to be italic

**Comment by mhaidarhanif** [comment]
@abidra need to resolve the requests from @r17x

**Comment by abidra** [comment]
> render

done

**Comment by mhaidarhanif** [comment]
@r17x can approve so this can be merged soon

File: src/content/reference/react-dom/components/input.md
Comment (mazipan) [inline src/content/reference/react-dom/components/input.md:53]: Pasang di sini terjemahan "mount", apa gak dipertahankan aja pakai mount di italic?

**Review by mazipan — APPROVED** [review]

File: src/content/reference/react-dom/components/input.md
Comment (mazipan) [reply src/content/reference/react-dom/components/input.md:53]: Oke

**Review by resir014 — APPROVED** [review]
Waiting on re-review from @r17x

**Review by r17x — APPROVED** [review]

**Pull request merged**
